tests/replicaset_test.go

```diff
@@ -145,7 +145,7 @@ var _ = Describe("[rfe_id:588][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 		Entry("[test_id:1406]to five, to six and then to zero replicas", 5, 6),
 	)
 
-	DescribeTable("[rfe_id:588][crit:medium][vendor:cnv-qe@redhat.com][level:component]should scale with scale subresource", func(startScale int, stopScale int) {
+	DescribeTable("[rfe_id:588][crit:medium][vendor:cnv-qe@redhat.com][level:component]should scale with scale subresource", decorators.Conformance, func(startScale int, stopScale int) {
```

Comment:
There were proposal to deprecate the ReplicaSet in favor of new Pool API, let's defer the Conformance for this.

Reply:
I will drop both replicaset commits.